### PR TITLE
Fixing list help text

### PR DESCRIPTION
**Pull request by avalcepina**

### CODE REVIEW

File: cmd/createenvironment.go [hunk 1]
```diff
@@ -38,7 +38,7 @@ var environmentCmd = &cobra.Command{
 	Short: "Create a new environment",
 	Long: AddAppName(`Create a new environment
     Example:
-    $AppName create environment env1 --organization org --file ./somepath.json`),
+    $AppName create environment env1 --organization org --file ./somepath.json --artifacts ./artifactspath.json`),
```
Comment (bgokden) [inline cmd/createenvironment.go]: Actually, this doesn't fix since it is a folder path and it expects yaml.

Comment (avalcepina) [reply cmd/createenvironment.go]: You mean it should be like this?

 create environment env1 --organization org --file ./somepath.json --artifacts ./artifactspath

Comment (bgokden) [reply cmd/createenvironment.go]: yes, it may be artifactsfolder

File: cmd/updateenvironment.go [hunk 2]
```diff
@@ -38,7 +38,7 @@ var updateenvironmentCmd = &cobra.Command{
 	Short: "Update a new environment",
 	Long: AddAppName(`Update a new environment
     Example:
-    $AppName update environment env1 --organization org --file ./somepath.json`),
+    $AppName update environment env1 --organization org --file ./somepath.json --artifacts ./artifactspath.json`),
```
Comment (bgokden) [inline cmd/updateenvironment.go]: also we have ./somepath.json but it should be yaml, currently only yaml is supported.

File: cmd/createenvironment.go [hunk 3]
```diff
@@ -38,7 +38,7 @@ var environmentCmd = &cobra.Command{
 	Short: "Create a new environment",
 	Long: AddAppName(`Create a new environment
     Example:
-    $AppName create environment env1 --organization org --file ./somepath.json --artifacts ./artifactspath.json`),
+    $AppName create environment env1 --organization org --file ./somepath.json --artifacts ./artifactsfolder`),
```
Comment (bgokden) [inline cmd/createenvironment.go]: Can you also fix this to something more meaningful and ending with yaml.
Like organizationdefinition.yaml

**Pull request merged**